src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -296,6 +297,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $event->setData($data);
             }
 
+            if (isset($data['features'])) {
```

Comment:
I understand that you are making sure there is no duplicates in `$data['features']` however I fail to understand the goal behind it. Maybe an helpful comment would help here ? 😄

Reply:
Done